src/subscriptions/SubscriptionsScreen.js

```diff
@@ -27,15 +38,30 @@ class SubscriptionsScreen extends React.Component {
     }
   };
 
+  handleFilterChange = (newFilter: string) => {
```

Comment:
If you change `newFilter` to `filter`, then you can simplify this to:
```
handleFilterChange = (filter: string) => this.setState({ filter });
```

In ES6, writing something like `{ filter }` is shorthand for `{ filter: filter }` where the first `filter` is the key of the object and the second `filter` is the variable that you are assigning as a value.